mmpose/apis/webcam/nodes/model_nodes/detector_node.py

```diff
@@ -124,7 +124,7 @@ def _post_process(self, preds) -> List[Dict]:
         """Post-process the predictions of MMDetection model."""
         instances = preds.pred_instances.cpu().numpy()
 
-        classes = self.model.dataset_meta['CLASSES']
+        classes = self.model.dataset_meta['classes']
```

Comment:
Is the field name updated in a recent mmdet version? Do we need to consider compatibility to both 'CLASSES' and 'classes'?

Reply:
Yes, MMDetection has changed all their metainfo keys from upper case to lower case. This modification does not lead to incompatibility in MMPose with metainfo key 'CLASSES' which is only used in metrics.